Task :exportEA FAILED because of  unknown property 'imageFormat'

**Describe the bug**
Using latest release 3.4.0 with dtcw to run task exportEA caused an failure 
>FAILURE: Build failed with an exception.
>
>* Where:
>Script 'C:\Users\spn2hi\.doctoolchain\docToolchain-3.4.0\scripts\exportEA.gradle' line: 200
>
>* What went wrong:
>Execution failed for task ':exportEA'.
>> Could not set unknown property 'imageFormat' for task ':exportEA' of type org.gradle.api.DefaultTask.

**To Reproduce**
Steps to reproduce the behavior:
1.Configure your project for exportEA action
2.Install docToolchain version 3.4.0
3.Execute task './dtcw exportEA'
4. See error

**Expected behavior**
The Enterprise Architect connection is opened, all diagrams found in the configured packages were exported in the expected image format (png if not explicitly set).

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Configuration**
 - docToolchain version: 3.4.0
 - OS: Windows Windows 10 22H2
  - Relevant SDK Versions JDK 1.17
dtcw 0.51 - ##DTCW_GIT_HASH##
docToolchain 3.4.0
OS/arch: MINGW64_NT-10.0-19045/x86_64
Available docToolchain environments: local
Environments with docToolchain [3.4.0]: local
Using environment: local
Using Java 17.0.8 [C:\Program Files\RedHat\java-17-openjdk\/bin/java]

--- a/changelog.adoc
+++ b/changelog.adoc
@@ -13,6 +13,7 @@ and this project tries to adhere to https://semver.org/spec/v2.0.0.html[Semantic
 * https://github.com/docToolchain/docToolchain/issues/1411[Jira links substitution is not working]
 * https://github.com/docToolchain/docToolchain/issues/1421[publishToConfluence fails with "No such property: attachmentPrefix for class: org.docToolchain.tasks.Asciidoc2ConfluenceTask"]
 * DTCW should preferably use DTC from the local installation and only rely on `+${PATH}+` otherwise
+* https://github.com/docToolchain/docToolchain/issues/1438[Task :exportEA FAILED because of unknown property 'imageFormat']
 
 === added
 

--- a/scripts/exportEA.gradle
+++ b/scripts/exportEA.gradle
@@ -153,6 +153,7 @@ task exportEA(
         def exportPath = ""
         def searchPath = ""
         def glossaryPath = ""
+        def imageFormat = ""
         def readme = """This folder contains exported diagrams or notes from Enterprise Architect.
 
 Please note that these are generated files but reside in the `src`-folder in order to be versioned.

